scripts/deno/ophan-components.ts

```diff
@@ -18,6 +18,30 @@ const html = await Promise.all(
 const known_errors = new Set([
 	// it only appears in DCR after hydration if signed in
 	'nav3 : topbar : my account',
+	'nav3 : topbar : account overview',
+	'nav3 : topbar : billing',
+	'nav3 : topbar : profile',
+	'nav3 : topbar : email prefs',
+	'nav3 : topbar : data privacy',
+	'nav3 : topbar : settings',
+	'nav3 : topbar : help',
+	'nav3 : topbar : comment activity',
+	'nav3 : topbar : sign out',
+
+	// Palette thrasher related
+	'container-0 | palette-styles-new-do-not-delete',
+	'palette-styles-new-do-not-delete',
+	'external | group-0 | card-@1',
+
+	// Can be fixed when we complete: https://github.com/guardian/dotcom-rendering/issues/7161
+	'all',
+
+	// These elements don't exist anymore in the DOM after Deeply Read component was added
+	'6 | text',
+	'7 | text',
+	'8 | text',
+	'9 | text',
+	'10 | text'
```

Comment:
New DOM: 

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/6ab8c7bc-a843-4d93-9c12-944c4e1939fc)

Reply:
Is **Deeply Read** only implemented on DCR? Why is there such a difference between the platforms?

Reply:
Yes it is only implemented on DCR. See relevant discussion here: https://chat.google.com/room/AAAA5nTgD0E/ugoyv0FxajM

And PRs:
https://github.com/guardian/frontend/pull/26189
https://github.com/guardian/dotcom-rendering/pull/7829